nix/commands/__load-config.sh

```diff
@@ -11,6 +11,7 @@ fi
 
 source "$config_path"
 
+declare -x ELECTRON_EXEC
```

Comment:
We don't want to export this into the user environment variables -- it's an internal variable used only by the `build-tools` scripts importing it.

Probably best to do something like

```sh
# used by scripts that source this file, so disable "unused variable" warnings
# shellcheck disable=SC2034
declare ELECTRON_EXEC
```